src/screens/contacts/contacts.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { View, Text, SafeAreaView, Button } from 'react-native';
+import { ListItem, Avatar, Overlay, Input } from 'react-native-elements';
+import { FlatList, ScrollView, TouchableOpacity } from 'react-native-gesture-handler';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import styles from './contacts.styles';
+
+import { _getUser } from '../../utils/asyncUtil';
+
+class ContactsScreen extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            allUsers: [],
+           
+            listUsers: []
+        }
+    }
+
+    async componentDidMount() {
+        let user = await _getUser()
+        this.convertUserObjectToArray(this.props.usersProfile, user);
+    }
+
+    convertUserObjectToArray = (usersObj = {}, own) => {
+        let usersArr = [];
+        Object.keys(usersObj).forEach((user = {}) => {
+            if (own !== user) {
+                usersArr.push({ ...usersObj[user], key: user });
+            }
+        });
+        this.setState({ allUsers: usersArr, listUsers: usersArr, mainUser: own });
+    }
+
+    listItem = ({ item }) => {
+     
+        if (!item) {
+            return null
+        }
+        return (
+            <ListItem key={item.name} bottomDivider>
+                <Avatar source={item.avatar} />
+                <ListItem.Content>
+                    <ListItem.Title>{item.name + ' (' + item.gender + ')'}</ListItem.Title>
```

Comment:
What if any item.gender comes null or undefined.?       App will crash

We can do it like this {item.gender && <ListItem.Title>{ListItem.Title}</ListItem.Title>}

Let's add this(i.e {item.name + ' (' + item.gender + ')'}) to 1 variable.

Reply:
Yes, But actually our data is local and all the form fields are mandatory that's why i skipped this part .